cpu/cortexm_common/cortexm_init.c

```diff
@@ -43,14 +47,18 @@ void cortexm_init(void)
 #if defined(CPU_ARCH_CORTEX_M3) || defined(CPU_ARCH_CORTEX_M4) || \
     defined(CPU_ARCH_CORTEX_M4F) || defined(CPU_ARCH_CORTEX_M7) || \
     (defined(CPU_ARCH_CORTEX_M0PLUS) && (__VTOR_PRESENT == 1))
+#ifndef SOFTDEVICE_PRESENT
     SCB->VTOR = (uint32_t)&_isr_vectors;
+#endif
 #endif
 
     /* initialize the interrupt priorities */
     /* set pendSV interrupt to same priority as the rest */
     NVIC_SetPriority(PendSV_IRQn, CPU_DEFAULT_IRQ_PRIO);
+#ifndef SOFTDEVICE_PRESENT
     /* set SVC interrupt to same priority as the rest */
     NVIC_SetPriority(SVCall_IRQn, CPU_DEFAULT_IRQ_PRIO);
+#endif
```

Comment:
Is this really necessary?

Reply:
RIOT is running wild if I call this line when I used softdevice. I didn't figure out why yet but I guess the softdevice does some black magic with this interrupt.

Reply:
Ok, let's think about a way to get it out of here. I suppose we can also re-setting it at the nrf52 cpu level.

Reply:
What about skipping the cortexm_init when softdevice is used ?

Reply:
You're totally right!

Reply:
Ok then, I'll try to push something like that :)